Web/Edubase.Web.UI/Web.config

```diff
@@ -53,7 +53,7 @@
     <add key="owin:appStartup" value="SASimulatorConfiguration" />
     <add key="SASimulatorGuid" value="" />
     <add key="SASimulatorUri" value="https://dfe-sign-in-simulator.azurewebsites.net/" />
-    <add key="TexunaApiBaseAddress" value="https://app-api-t1dv-edubase.azurewebsites.net/edubase/rest/" />
+    <add key="TexunaApiBaseAddress" value="https://app-api-t1dv-edubase-sandbox1.azurewebsites.net/edubase/rest/" />
```

Comment:
You should keep this as a local change and not commit it

Reply:
resolved